### PR TITLE
Update blockly to use Circle 2.0

### PR DESCRIPTION
**Pull request by balderdash**
Mostly just used the auto-generated config and removed the parts we didn't need.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/code-dot-org/blockly
+    parallelism: 1
+    docker:
+    - image: circleci/node:6.13
```
Comment (balderdash) [inline .circleci/config.yml]: Circle offers a set of docker images, but they didn't have one at node 6.9, only 6.13 (see: https://hub.docker.com/r/circleci/node/tags/).

We could create our own image like Will did for the main repo to get the right version, but I'm not sure that's worth the effort.

Comment (Hamms) [reply .circleci/config.yml]: Agreed; 6.13 should be fine

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/code-dot-org/blockly
```
Comment (Hamms) [inline .circleci/config.yml]: why not just `blockly`?

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    parallelism: 1
+    docker:
+    - image: circleci/node:6.13
+    steps:
+    - checkout
+    - restore_cache:
+        keys:
+        - v2-dep-{{ .Branch }}-
```
Comment (Hamms) [inline .circleci/config.yml]: should this also have `{{ epoch }}` like `save_cache`?

Comment (Hamms) [reply .circleci/config.yml]: and actually, shouldn't both be using something like `checksum "package.json"`?

**Comment by balderdash** [comment]
Yup! The stripped-down container loads quicker than the old circle 1.0 environment.

**Pull request merged**